PowerShell/JumpCloud Module/Public/Users/New-JCUser.ps1

```diff
@@ -405,6 +410,20 @@ Function New-JCUser ()
             $body.Add('mfa', $mfa)
         }
 
+        if ($suspended -eq $true) {
+            $body['state'] = 'SUSPENDED'
```

Comment:
If suspended is true, then state shouldn't state be calculated/ set from our API?